src/main/java/eu/dissco/core/digitalspecimenprocessor/service/ProcessingService.java

```diff
@@ -337,7 +340,7 @@ private Set<DigitalSpecimenRecord> createNewDigitalSpecimen(List<DigitalSpecimen
       }
       log.info("Successfully created {} new digitalSpecimen", digitalSpecimenRecords.size());
       return digitalSpecimenRecords.keySet();
-    } catch (IOException e) {
+    } catch (IOException | ElasticsearchException e) {
```

Comment:
Should you not apply the same logic to updates?

Reply:
Added them at the other places.